src/components/ebay-avatar/index.marko

```diff
@@ -0,0 +1,41 @@
+import { processHtmlAttributes } from "../../common/html-attributes";
+import { getColorForText } from "./util";
+static {
+    var ignoredAttributes = [
+        "a11yText",
+        "class",
+        "size",
+        "color",
+        "img",
+        "username",
+    ];
+}
+$ var isSignedOut = !input.username;
+$ var color = getColorForText(input.username, input.color);
+
+<div
+    ...processHtmlAttributes(input, ignoredAttributes)
+    role="img"
+    aria-label=input.a11yText
+    class=[
+        "avatar",
+        input.class,
+        input.size && `avatar--${input.size}`,
+        `avatar--${color}`,
+    ]
+>
+    <if(input.img)>
+        <img ...input.img>
+    </if>
+
+    <else-if(input.renderBody)>
+        <${input}/>
+    </else-if>
+    <else-if(!isSignedOut)>
```

Comment:
With this conditional structure refactor it might be simpler if you remove the `isSignedOut` variable entirely and just check `else-if(input.username)`

Reply:
Good catch, its easy to miss simple things like this.